jdi-light-mobile/src/main/java/com/epam/jdi/light/mobile/driver/MobileDriverData.java

```diff
@@ -0,0 +1,28 @@
+package com.epam.jdi.light.mobile.driver;
+
+import com.epam.jdi.tools.func.JAction1;
+import org.openqa.selenium.MutableCapabilities;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Created by Roman Iovlev on 26.09.2019
+ * Email: roman.iovlev.jdi@gmail.com; Skype: roman.iovlev
+ */
+public class MobileDriverData {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The utility class name 'MobileDriverData' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper)'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5217753)